.travis.yml

```diff
@@ -5,7 +5,9 @@ php:
   - 5.4
   - 5.5
   - 5.6
-  - 7
+  - 7.0
+  - 7.1
+  - nightly
```

Comment:
Will this library continue to work once a nightly contains breaking changes or PHP 8 or PHP 9 will be released? I'm not sure we can guarantee this, so I would rather remove the "nightly" build here.

Also, I would vote to not merge this yet at least until the issues with PHP 7.1 are resolved :+1: